ci-operator/config/openshift/microshift/openshift-microshift-main.yaml

```diff
@@ -84,6 +84,15 @@ tests:
       EC2_AMI: ami-02b8534ff4b424939
       EC2_INSTANCE_TYPE: c5n.metal
     workflow: openshift-microshift-e2e-metal-tests
+- as: microshift-metal-cache
+  cluster: build02
+  cron: 0 0 * * *
+  steps:
+    cluster_profile: aws
+    env:
+      EC2_AMI: ami-02b8534ff4b424939
+      EC2_INSTANCE_TYPE: c5n.metal
```

Comment:
Looking at the commands in the job, do we need to run it in a metal instance?

Reply:
Switched to t3.2xlarge by default